app/scripts/modules/cloudfoundry/src/pipeline/stages/mapLoadBalancers/cloudfoundryMapLoadBalancersStage.module.ts

```diff
@@ -18,6 +18,6 @@ Registry.pipeline.registerStage({
     { type: 'requiredField', preventSave: true, fieldName: 'credentials', fieldLabel: 'account' },
     { type: 'requiredField', preventSave: true, fieldName: 'region' },
     { type: 'requiredField', preventSave: true, fieldName: 'target' },
-    { type: 'cfRequiredRoutesField', preventSave: true, fieldName: 'loadBalancerNames' },
+    //{ type: 'cfRequiredRoutesField', preventSave: true, fieldName: 'loadBalancerNames' },
```

Comment:
any reason why we want to leave this commented rather than just deleting it?

Reply:
deleted